client.js

```diff
@@ -1,7 +1,7 @@
-import { sha256 } from 'js-sha256';
 import { Submarine } from './src/submarine';
 
 // Define a customer, shop and API secret for use in development only.
+// Include an endpoint in the API_ENDPOINTS that corresponds to your local instance of Submarine.
```

Comment:
This is specific to Disco developers, so I don't think it should be included in this public repo.